### ISSUE TITLE
Can't update old post (editor V1)

### ISSUE THREAD

**Issue by equiman** [forem/forem#18348]
[x] done

**Describe the bug**

I'm trying to update one of my oldest posts that was created with editor V1. But when I hit the [save] button got this message:

```
Whoops, something went wrong:
published_at: updating published_at for articles that have already been published is not allowed
```

**To Reproduce**

1. Go to https://dev.to/equiman/setup-macos-for-development-3kc2
2. Click on edit option
3. Add this markdown on `## Git` section

```markdown
### Auto Setup Remote

If you want to avoid to use `git push --set-upstream...` message.

git config --global push.autoSetupRemote true

### Use Git Alias

{% post https://dev.to/equiman/zsh-cheatsheet-for-git-plugin-1f6a %}
```
4. Hit the [save] button

**Expected behavior**

Save the update post and update.

**Screenshots**

![image](https://user-images.githubusercontent.com/933393/185149414-c5c35a25-0e57-44ae-a397-7a5107a05d51.png)

**Desktop (please complete the following information):**

- OS, version: Win10
- Browser, version: Version 104.0.5112.81 (Official Build) (64-bit)

**Additional context**

Preview mode is working and showing those changes

![image](https://user-images.githubusercontent.com/933393/185151015-8dacda38-17ff-45c6-a018-75f81570106f.png)

**Comment by ApayRus** [comment]
I have the same issue. Did you solve it?

**Comment by jackblender2323** [comment]
Update your WordPress to the latest version 9.0 which was released recently. My client is also facing this issue while update in their [insta thunder](https://instamods.io/insta-thunder-apk/). The New version is awesome must try it.

**Comment by equiman** [comment]
@benhalpern can some one deletet the previous dangerous messages?

**Comment by halvinterry687** [comment]
I recommend updating your WordPress to the latest version 9.0, which was recently released. My client encountered a similar issue while updating their [best one wheel boards](https://erideshub.com/best-onewheel-boards/), and upgrading to the new version resolved the problem. The latest release comes with impressive features and is highly recommended for use. Therefore, I suggest giving it a try.

**Comment by villerpeter** [comment]
Updating old posts on WordPress is a great way to keep your website's content fresh and relevant. To update an old post on WordPress, you can follow these steps:

Log in to your WordPress dashboard and navigate to the post you want to update.

Click on the "Edit" button to open the post editor.

Make the necessary changes to the post, such as updating the text, adding new images or videos, or adjusting the formatting.

Once you have made your changes, click on the "Update" button to save the updated post.

In addition to updating your post, you can also consider adding an orange speedtest to the content. An orange speedtest is a tool that measures the speed of your internet connection, and it can be a helpful addition to a post that discusses internet-related topics. You can easily embed an [orange speedtest](https://speedtestorange.com/) into your post by using a plugin or copying and pasting the embed code provided by a speedtest website. By adding an orange speedtest to your post, you can provide your readers with a valuable resource while also improving the overall quality of your content.

**Comment by halverurex34** [comment]
Yes, of course! You can always update your old post and add new information or examples to it, including mentioning the [Prisma technology](https://www.prismaimaging.com/technology/). Just make sure that you are providing accurate and relevant information that adds value to the post and is consistent with the topic and context.

If you are updating a post on a platform that allows editing, such as a blog or a forum, you can simply go to the post and edit it by adding the new information or example. If the platform does not allow editing, you can reply to your original post with the new information or example, or create a new post altogether.

Remember to be transparent about your updates, especially if the post has been viewed or shared by others. You can add a note or disclaimer at the beginning or end of the post indicating that it has been updated with new information or example. This will help your readers understand the context and relevance of your post.

**Comment by mikeydorje** [comment]
Code review suggests this may be resolved or no longer applicable due to changes in this area since the issue was filed. Closing due to age and low activity. Please reopen with current repro steps if this persists.
